Jamfile

```diff
@@ -27,7 +27,6 @@ if $(BOOST_ROOT)
 else
 {
 	local boost-lib-search-path =
-		<search>/usr/lib
```

Comment:
yes, and in fact, it gets in the way, since it's explicitly passed, and it takes higher priority, event if a custom toolchain does not depend on the system libraries